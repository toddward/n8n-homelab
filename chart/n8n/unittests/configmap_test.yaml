suite: test configmap

templates:
  - configmap.yaml

tests:
  - it: should match snapshot of default values
    release:
      name: n8n
      namespace: n8n
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }
