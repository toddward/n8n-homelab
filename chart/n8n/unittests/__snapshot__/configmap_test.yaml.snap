should match snapshot of default values:
  1: |
    apiVersion: v1
    data:
      DB_SQLITE_VACUUM_ON_STARTUP: "false"
      DB_TYPE: sqlite
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-database-configmap
  2: |
    apiVersion: v1
    data:
      N8N_LOG_LEVEL: info
      N8N_LOG_OUTPUT: console
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-logging-configmap
  3: |
    apiVersion: v1
    data:
      N8N_DIAGNOSTICS_CONFIG_BACKEND: ""
      N8N_DIAGNOSTICS_CONFIG_FRONTEND: ""
      N8N_DIAGNOSTICS_ENABLED: "false"
      N8N_DIAGNOSTICS_POSTHOG_API_HOST: ""
      N8N_DIAGNOSTICS_POSTHOG_API_KEY: ""
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-diagnostics-configmap
  4: |
    apiVersion: v1
    data:
      N8N_VERSION_NOTIFICATIONS_ENABLED: "false"
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-version-notifications-configmap
  5: |
    apiVersion: v1
    data:
      N8N_PUBLIC_API_DISABLED: "false"
      N8N_PUBLIC_API_ENDPOINT: api
      N8N_PUBLIC_API_SWAGGERUI_DISABLED: "false"
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-public-api-configmap
  6: |
    apiVersion: v1
    data:
      N8N_WORKFLOW_HISTORY_ENABLED: "true"
      N8N_WORKFLOW_HISTORY_PRUNE_TIME: "336"
    kind: ConfigMap
    metadata:
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-workflow-history-configmap
